pygen/pygen_src/riscv_instr_pkg.py

```diff
@@ -1,1084 +1,1207 @@
+    HPMCOUNTER19H = 0xC93
+    HPMCOUNTER20H = 0xC94
+    HPMCOUNTER21H = 0xC95
+    HPMCOUNTER22H = 0xC96
+    HPMCOUNTER23H = 0xC97
+    HPMCOUNTER24H = 0xC98
+    HPMCOUNTER25H = 0xC99
+    HPMCOUNTER26H = 0xC9A
+    HPMCOUNTER27H = 0xC9B
+    HPMCOUNTER28H = 0xC9C
+    HPMCOUNTER29H = 0xC9D
+    HPMCOUNTER30H = 0xC9E
+    HPMCOUNTER31H = 0xC9F
+    SSTATUS = 0x100
+    SEDELEG = 0x102
+    SIDELEG = 0x103
+    SIE = 0x104
+    STVEC = 0x105
+    SCOUNTEREN = 0x106
+    SSCRATCH = 0x140
+    SEPC = 0x141
+    SCAUSE = 0x142
+    STVAL = 0x143
+    SIP = 0x144
+    SATP = 0x180
+    MVENDORID = 0xF11
+    MARCHID = 0xF12
+    MIMPID = 0xF13
+    MHARTID = 0xF14
+    MSTATUS = 0x300
+    MISA = 0x301
+    MEDELEG = 0x302
+    MIDELEG = 0x303
+    MIE = 0x304
+    MTVEC = 0x305
+    MCOUNTEREN = 0x306
+    MSCRATCH = 0x340
+    MEPC = 0x341
+    MCAUSE = 0x342
+    MTVAL = 0x343
+    MIP = 0x344
+    PMPCFG0 = 0x3A0
+    PMPCFG1 = 0x3A1
+    PMPCFG2 = 0x3A2
+    PMPCFG3 = 0x3A3
+    PMPADDR0 = 0x3B0
+    PMPADDR1 = 0x3B1
+    PMPADDR2 = 0x3B2
+    PMPADDR3 = 0x3B3
+    PMPADDR4 = 0x3B4
+    PMPADDR5 = 0x3B5
+    PMPADDR6 = 0x3B6
+    PMPADDR7 = 0x3B7
+    PMPADDR8 = 0x3B8
+    PMPADDR9 = 0x3B9
+    PMPADDR10 = 0x3BA
+    PMPADDR11 = 0x3BB
+    PMPADDR12 = 0x3BC
+    PMPADDR13 = 0x3BD
+    PMPADDR14 = 0x3BE
+    PMPADDR15 = 0x3BF
+    MCYCLE = 0xB00
+    MINSTRET = 0xB02
+    MHPMCOUNTER3 = 0xB03
+    MHPMCOUNTER4 = 0xB04
+    MHPMCOUNTER5 = 0xB05
+    MHPMCOUNTER6 = 0xB06
+    MHPMCOUNTER7 = 0xB07
+    MHPMCOUNTER8 = 0xB08
+    MHPMCOUNTER9 = 0xB09
+    MHPMCOUNTER10 = 0xB0A
+    MHPMCOUNTER11 = 0xB0B
+    MHPMCOUNTER12 = 0xB0C
+    MHPMCOUNTER13 = 0xB0D
+    MHPMCOUNTER14 = 0xB0E
+    MHPMCOUNTER15 = 0xB0F
+    MHPMCOUNTER16 = 0xB10
+    MHPMCOUNTER17 = 0xB11
+    MHPMCOUNTER18 = 0xB12
+    MHPMCOUNTER19 = 0xB13
+    MHPMCOUNTER20 = 0xB14
+    MHPMCOUNTER21 = 0xB15
+    MHPMCOUNTER22 = 0xB16
+    MHPMCOUNTER23 = 0xB17
+    MHPMCOUNTER24 = 0xB18
+    MHPMCOUNTER25 = 0xB19
+    MHPMCOUNTER26 = 0xB1A
+    MHPMCOUNTER27 = 0xB1B
+    MHPMCOUNTER28 = 0xB1C
+    MHPMCOUNTER29 = 0xB1D
+    MHPMCOUNTER30 = 0xB1E
+    MHPMCOUNTER31 = 0xB1F
+    MCYCLEH = 0xB80
+    MINSTRETH = 0xB82
+    MHPMCOUNTER3H = 0xB83
+    MHPMCOUNTER4H = 0xB84
+    MHPMCOUNTER5H = 0xB85
+    MHPMCOUNTER6H = 0xB86
+    MHPMCOUNTER7H = 0xB87
+    MHPMCOUNTER8H = 0xB88
+    MHPMCOUNTER9H = 0xB89
+    MHPMCOUNTER10H = 0xB8
+    MHPMCOUNTER11H = 0xB8
+    MHPMCOUNTER12H = 0xB8
+    MHPMCOUNTER13H = 0xB8
+    MHPMCOUNTER14H = 0xB8
+    MHPMCOUNTER15H = 0xB8
+    MHPMCOUNTER16H = 0xB9
+    MHPMCOUNTER17H = 0xB9
+    MHPMCOUNTER18H = 0xB9
+    MHPMCOUNTER19H = 0xB9
+    MHPMCOUNTER20H = 0xB9
+    MHPMCOUNTER21H = 0xB9
+    MHPMCOUNTER22H = 0xB9
+    MHPMCOUNTER23H = 0xB9
+    MHPMCOUNTER24H = 0xB9
+    MHPMCOUNTER25H = 0xB9
+    MHPMCOUNTER26H = 0xB9
+    MHPMCOUNTER27H = 0xB9
+    MHPMCOUNTER28H = 0xB9
+    MHPMCOUNTER29H = 0xB9
+    MHPMCOUNTER30H = 0xB9
+    MHPMCOUNTER31H = 0xB9
```

Comment:
I think these should be `0xB8A` through `0xB9F` ?

Reply:
Yes